CoreDataCache+CoreDataClass.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CoreDataCache+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by Dennis Nehrenheim on 19.05.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(CoreDataCache)
+public class CoreDataCache: NSManagedObject {
```

Comment:
`CoreDataCache` is an implementation detail of `CoreDataFeedStore` thus it should not be public.
Also it is not intended to be inherited thus it should be `final`

Please remove `public` and add `final` modifier.

```suggestion
final class CoreDataCache: NSManagedObject {
```